standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5144,6 +5159,7 @@ private int abortTxns(Connection dbConn, List<Long> txnids, boolean checkHeartbe
       if (MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.METASTORE_ACIDMETRICS_EXT_ON)) {
         Metrics.getOrCreateCounter(MetricsConstants.TOTAL_NUM_ABORTED_TXNS).inc(txnids.size());
       }
+      LOG.info("Txns are aborted successfully due to : {}", errorMsg);
```

Comment:
maybe warn?

Reply:
For `performTimeOuts()`
Removed this log -  `LOG.info("Aborted the following transactions due to timeout: {}", batchToAbort.toString());`

For `commitTxn()`
Changed this log to include only the committed transaction details and not the ones that are going to be aborted.

Added aborted transaction IDs in the generic log.